src/middleware/node/index.ts

```diff
@@ -0,0 +1,18 @@
+import { Webhooks } from "../../index";
+import { EmitterWebhookEvent } from "../../types";
+import { middleware } from "./middleware";
+import { onUnhandledRequestDefault } from "./on-unhandled-request-default";
+import { MiddlewareOptions } from "./types";
+
+export function createNodeMiddleware(
+  webhooks: Webhooks<EmitterWebhookEvent>,
```

Comment:
@G-Rath @oscard0m  @wolfy1339 sorry I think we discussed this recently, I've been working on other parts of @octokit the past few weeks. Is there a reason why I have to set a type if I want to use the `Webhooks` constructor in TypeScript code? I don't think that should be the case. Is there a reason why we don't set a default?

Reply:
This seems to have been an oversight in #494, it was defaulted to `unknown` before v8

Reply:
created a PR here #514